provider/resource_server.go

```diff
@@ -246,19 +135,11 @@ func applyServer(s *online.Server, rpns map[string]*online.RPNv2, d *schema.Reso
 			})
 
 		case online.Private:
-			for _, p := range d.Get("private_interface").([]interface{}) {
-				m := p.(map[string]interface{})
-				m["mac"] = strings.ToLower(iface.MAC)
-				m["vlan_id"] = 0
-
-				rpn, ok := rpns[m["rpn"].(string)]
-				if ok {
-					m["vlan_id"] = rpn.MemberByServerID(s.ID).VLAN
-				}
-
-				private = append(private, m)
-			}
-
+			private = append(private, map[string]interface{}{
+				"mac":     strings.ToLower(iface.MAC),
+				"dns":     iface.Reverse,
```

Comment:
Why does the private iface have a reverse?

Reply:
It will be empty, but since it is defined in the schema I preferred to set it

Reply:
okay! it it's in the schema let's add it here too then